ide-gen-proto.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+
+# This script is used to generate protobuf files for all services.
+# Useful to ensure code can compile without Docker, and provide hints for IDEs.
+# Several dev tools including: cargo, protoc, python grpc_tools.protoc, and rebar3 may be required to run this script.
+
+base_dir=$(pwd)
+
+gen_proto_dotnet() {
+  echo "Generating .NET protobuf files for $1"
+  cd "$base_dir"/src/"$1" || return
+  mkdir -p ./src/protos/
+  cp -r "$base_dir"/pb/ ./src/protos/
+  cd "$base_dir" || return
+}
+
+gen_proto_elixir() {
+  echo "Generating Elixir protobuf files for $1"
+  cd "$base_dir"/src/"$1" || return
+  cp "$base_dir"/pb/demo.proto ./proto/demo.proto
+  rebar3 grpc gen
+  cd "$base_dir" || return
+}
+
+gen_proto_go() {
+  echo "Generating Go protobuf files for $1"
+  cd "$base_dir"/src/"$1" || return
+  protoc -I ../../pb ./../../pb/demo.proto --go_out=./ --go-grpc_out=./
+  cd "$base_dir" || return
+}
+
+gen_proto_js() {
+  echo "Generating Javascript protobuf files for $1"
+  cd "$base_dir"/src/"$1" || return
+  cp "$base_dir"/pb/demo.proto .
+  cd "$base_dir" || return
+}
+
+gen_proto_python() {
+  echo "Generating Python protobuf files for $1"
+  cd "$base_dir"/src/"$1" || return
+  python -m grpc_tools.protoc -I=../../pb --python_out=./ --grpc_python_out=./ ./../../pb/demo.proto
```

Comment:
Not sure if that will be the case for most users, but I had to change this to `python3` in order to work.